### PR TITLE
Setup database (RethinkDB)

### PR DESCRIPTION
**Pull request by gmittal**
Setup RethinkDB.

### CODE REVIEW

**Comment by gmittal** [comment]
@ankitr PTAL

**Comment by gmittal** [comment]
Closes #9

File: app.js [hunk 1]
```diff
@@ -19,8 +19,26 @@ app.use(function(req, res, next) { // enable CORS and assume JSON return structu
   next();
 });
 var hash = new hashid(process.env.HASH_SALT);
-var db = new loki(__dirname+'/main.json'); // intiialize datastore
-var pending_requests = db.addCollection("pending_requests"); // pending requests
+
+var connection = null;
```
Comment (ankitr) [inline app.js:23]: Is this idiomatic? Globals are icky, but I get if it's how things are done.

**Comment by ankitr** [comment]
Sans question, LGTM.

File: app.js (hunk 1 above)
Comment (gmittal) [inline app.js:23]: That's how Rethink's documentation explains it. 

Basically the advantage to storing the connection from Rethink in a global connection variable is so you don't need to reinitialize the database connection each time you want to query it.

**Pull request merged**